pkgs/development/ocaml-modules/ocaml-protoc-plugin/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, fetchFromGitHub
+, buildDunePackage
+, pkg-config
+, protobuf
+, zarith
+, ppx_deriving
+, ppx_deriving_yojson
+, re
+, dune-site
+, ppx_expect
+}:
+
+buildDunePackage rec {
+  pname = "ocaml-protoc-plugin";
+  version = "4.3.1";
+
+  duneVersion = "3";
+
+  minimalOcamlVersion = "4.08";
+
+  src = fetchFromGitHub {
+    owner = "issuu";
+    repo = pname;
+    rev = version;
+    hash =
+      "sha256-KFd43Ukz5gMeM3ik2VlfaIPwcZe9yaxk9VcQIrauUGU=";
+
+  };
+
+  nativeBuildInputs =
+    [ pkg-config protobuf ];
+  buildInputs = [
+    zarith
+    ppx_deriving
+    ppx_deriving_yojson
+    re
+    dune-site
+    ppx_expect
+    protobuf
+  ];
+  doCheck = true;
+  nativeCheckInputs = [ protobuf ];
```

Comment:
AFAIK, `nativeCheckInputs` does not exist and can safely be removed.

Reply:
It exists and is used, but probably doesn't have to be duplicated between this and nativeBuildInputs

Reply:
Indeed, I just checked. Yesterday, when I did the search, I got no results. Probably VS Code was having an issue.

Therefore, ignore my feedback here.